packages/iov-bns/src/types.ts

```diff
@@ -342,6 +342,69 @@ export interface Decoder<T extends {}> {
   readonly decode: (data: Uint8Array) => T;
 }
 
+// Transactions: Term Deposit
+
+export interface TermDepositBonus {
+  readonly lockingPeriod: number;
```

Comment:
`lockingPeriod` should be `lockinPeriod` (without the "g").  We want depositors to "lock-in" their tokens.